### DISCUSSION TITLE
Welcome to laravel-google-calendar Discussions!

### DISCUSSION THREAD

**Discussion by anggadwi24** [anggadwi24/laravel-google-calendar#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aldiferdian** [comment]
tutor buat console bang

**Reply by anggadwi24** [reply]
bisa langsung email : anggadwi1014@gmail.com

**Reply by aldiferdian** [reply]
makasi kak

**Reply by anggadwi24** [reply]
siap ^_^
